Resources/Private/CodeTemplates/Extbase/Configuration/ExtensionBuilder/settings.yamlt

```diff
@@ -49,6 +49,7 @@ overwriteSettings:
   Configuration:
     #TCA merge not possible - use overrides directory
     #TypoScript: keep
+    #GIT: false
```

Comment:
I think this should not be under Configuration. Since we are in the overwrite settings, Configuration means the folder named "Configuration".
I wpuld suggest it should look like this (on the first level of overwriteSettings),
with an explaining comment und commented by default

`overwriteSettings:`
`    # uncomment if you don't want an empty git repository to be created`
`    #.git: skip`

Reply:
Indeed it's a better approach to have it enable per default to motivate devs to use git. Until the option is available, it's all good